man/systemctl.xml

```diff
@@ -1178,12 +1178,17 @@ kobject-uevent 1 systemd-udevd-kernel.socket systemd-udevd.service
                   </row>
                   <row>
                     <entry><literal>disabled</literal></entry>
-                    <entry>Unit file is not enabled, but contains an <literal>[Install]</literal> section with installation instructions.</entry>
+                    <entry>The unit file is not enabled, but contains an <literal>[Install]</literal> section with installation instructions.</entry>
                     <entry>&gt; 0</entry>
                   </row>
                   <row>
+                    <entry><literal>generated</literal></entry>
+                    <entry>The unit file was generated dynamically via a generator tool. See <citerefentry><refentrytitle>systemd.generator</refentrytitle><manvolnum>7</manvolnum></citerefentry>.</entry>
+                    <entry>0</entry>
```

Comment:
I'd like to have some freedom there for now, to change things like the ordering of the rules a bit. The past tells us that the precise definition of the terms has changed a bit, and the order of precedence. Documenting this too closely means we cannot alter things anymore that easily...